<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-JT28CA7" failures="2" tests="3" name="Test" time="0.098" errors="0" timestamp="2024-09-10T06:43:34 IST">
  <testcase classname="testNG.FailedTestCases" name="tc1" time="0.015">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testNG.FailedTestCases.tc1(FailedTestCases.java:13)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- tc1 -->
  <testcase classname="testNG.FailedTestCases" name="tc2" time="0.001"/>
  <testcase classname="testNG.FailedTestCases" name="tc3" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testNG.FailedTestCases.tc3(FailedTestCases.java:25)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- tc3 -->
</testsuite> <!-- Test -->
